pkg/async-gateway/main.go

```diff
@@ -19,60 +19,28 @@ package main
 import (
 	"flag"
 	"net/http"
-	"os"
-	"strings"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
+
+	"github.com/cortexlabs/cortex/pkg/async-gateway/exit"
+	"github.com/cortexlabs/cortex/pkg/lib/aws"
+	cr "github.com/cortexlabs/cortex/pkg/lib/configreader"
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/lib/logging"
+	"github.com/cortexlabs/cortex/pkg/lib/telemetry"
+	"github.com/cortexlabs/cortex/pkg/types/clusterconfig"
+	"github.com/cortexlabs/cortex/pkg/types/userconfig"
 )
 
 const (
-	_defaultPort = "8080"
+	_defaultPort       = "8080"
+	_clusterConfigPath = "/configs/cluster/cluster.yaml"
```

Comment:
this should not be hard coded, has to be passed as a flag

Reply:
all the other flags must be modified accordingly as well

Reply:
Right, this makes sense. Shouldn't this also apply to the operator? It was done in the light of the operator.